packages/block-library/src/loginout/edit.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * WordPress dependencies
+ */
+import { PanelBody, ToggleControl, Disabled } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { InspectorControls, useBlockProps } from '@wordpress/block-editor';
+import ServerSideRender from '@wordpress/server-side-render';
+
+export default function ArchivesEdit( { attributes, setAttributes } ) {
```

Comment:
I suggest renaming "ArchivesEdit" to something more suitable for this block.

Reply:
Darned copy-paste.. 🤦 
Fixed. Thanks for catching this!